iOS/UI/Manga/MangaViewController.swift

```diff
@@ -1160,6 +1160,21 @@ extension MangaViewController: MangaDetailHeaderViewDelegate {
         present(SFSafariViewController(url: url), animated: true)
     }
 
+    // copy manga link when holding down the web view button
+    func safariHeld() {
+        guard
+            let url = manga.url
+        else { return }
+        UIPasteboard.general.string = url.absoluteString
+        let alert = UIAlertController(
+                   title: NSLocalizedString("LINK_COPIED", comment: ""),
+                   message: NSLocalizedString("LINK_COPIED_TEXT", comment: ""),
+                   preferredStyle: .alert
+               )
+               alert.addAction(UIAlertAction(title: "OK", style: .default, handler: nil))
+               present(alert, animated: true)
+    }
```

Comment:
indentation still looking a little off, also make sure to localize "OK" and use cancel style
```suggestion
        guard let url = manga.url else { return }
        UIPasteboard.general.string = url.absoluteString
        let alert = UIAlertController(
            title: NSLocalizedString("LINK_COPIED", comment: ""),
            message: NSLocalizedString("LINK_COPIED_TEXT", comment: ""),
            preferredStyle: .alert
        )
        alert.addAction(UIAlertAction(title: NSLocalizedString("OK", comment: ""), style: .cancel))
        present(alert, animated: true)
    }
```